[#62050] Consolidate "Download PDF" feature with the "Generate PDF" f… (#18237)
opf/openproject: docs/user-guide/work-packages/exporting/README.md

![Single work package export options in OpenProject](openproject_user_guide_work_package_export_options.png)

### Export single work package in PDF format

If you select **Generate PDF**, a modal will open, where you can adjust the following: 

- **Template** is a dropdown menu showing all of the options currently enabled. At moment possible template options include:

   - *Attributes and description* - this template lists all the work package attributes [configured in the work package form](../../../system-admin-guide/manage-work-packages/work-package-types/#work-package-form-configuration-enterprise-add-on), regardless whether they are filled out or not.
   - *Contract* - this template includes work package details formatted to the standard German contract form.

    > [!TIP] You can define which templates are enabled for specific work package types in the [administration settings](../../../system-admin-guide/manage-work-packages/work-package-types).

- **Hyphenation** - if selected, a break line will be included into the export between word for improved layout.

> - **Language and hyphenation** - a dropdown menu showing languages, available for activation in OpenProject. Selected language serves as a base for hyphenation. It does not change the language used in the PDF export.

Reviewer: ```suggestion
- **Language and hyphenation** - a dropdown menu showing languages to be used for hyphenation. The selection does not change the language used in the PDF export.
```
